cli/lib/exec/open.js

```diff
@@ -35,9 +36,7 @@ module.exports = {
       args.push('--project', options.project)
     }
 
-    if (isComponentTesting) {
-      args.push('--testing-type', 'component')
-    }
+    processTestingType(options.testingType, args)
```

Comment:
Instead of mutating the args in this function, maybe we could return the new arg then push it in?

```js
const testingTypeArg = processTestingType(options.testingType, args)
args.push(testingTypeArg)
```

I find these side-effects (eg, mutating an function argument that is passed by reference) can be difficult to follow - you need to dive into the function and keep a mental model of what and where things are mutated.